tests/pages/firefox/new.py

```diff
@@ -14,10 +14,15 @@ class FirefoxNewPage(FirefoxBasePage):
     _download_button_locator = (By.CSS_SELECTOR, '#features-download .download-button .download-link')
     _thank_you_message_locator = (By.ID, 'scene2-main')
 
-    def download_firefox(self):
+    @property
+    def is_download_button_displayed(self):
         els = [el for el in self.find_elements(self._download_button_locator) if el.is_displayed()]
         assert len(els) == 1, 'Expected one download button to be displayed'
-        els[0].click()
+        return els[0]
```

Comment:
A property prefixed `is_` should really return a boolean. We could create a private `_download_button` property that can be used from both `is_download_button_displayed` and `download_firefox`.